server/comment/hooks.js

```diff
@@ -39,13 +39,19 @@ async function sendEmailNotification(comment, db) {
     ]
   });
   const { author, repository, activity } = comment;
+  const lastFourComments = await activity.getComments({
```

Comment:
Define this more generic like bellow (`previousComments`). Use offset to eliminate 1.